.github/workflows/build-images.yaml

```diff
@@ -230,6 +236,17 @@ jobs:
           build-args: |
             GO_BASE_IMG=${{ env.GO_BASE_IMG }}
 
+      # - name: Build and push neonvm-daemon image
+      #   uses: docker/build-push-action@v3
+      #   with:
+      #     context: .
+      #     platforms: linux/amd64
+      #     push: true
+      #     file: neonvm/daemon/Dockerfile
+      #     tags: ${{ needs.tags.outputs.daemon }}
+      #     build-args: |
+      #       GO_BASE_IMG=${{ env.GO_BASE_IMG }}
```

Comment:
Should we remove this, and all the other references to `[neonvm-]daemon` in this workflow?